### PR TITLE
Restructure the Basics Content

### PR DESCRIPTION
**Pull request by mgrover1**
Add a new section at the beginning of the book

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 format: jb-book
 root: README
 parts:
-  - caption: Foundations
+  - caption: Radar Weather Foundations
```
Comment (mgrover1) [inline _toc.yml:4]:
```suggestion
  - caption: Weather Radar Foundations
```

File: _toc.yml [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+    chapters:
+      - file: notebooks/radar-basics/radar-basics
+  - caption: Radar Toolkit Foundations
```
Comment (mgrover1) [inline _toc.yml:7]:
```suggestion
  - caption: Radar Software Foundations
```

**Comment by jukent** [comment]
Just fixed a git conflict in the toc!

**Comment by brian-rose** [comment]
A cosmetic problem with the new secrets-enabled workflows is that the preview link appears long before a valid preview is available. We can probably tweak that behavior.

**Pull request merged**